src/legacy/components/Receive.tsx

```diff
@@ -203,6 +199,16 @@ const AddressBlock = ({
                   />
                 </button>
               )}
+
+              <button
+                className={[cstyles.primarybutton, styles.buttomMarginTop].join(
+                  ' ',
+                )}
+                type='button'
+                onClick={() => fetchAndSetSinglePrivKey(address, 'paperWallet')}
+              >
+                Turn address into paper wallet
```

Comment:
Can we change it to "Generate paper wallet" please?

Reply:
@br4in3x it has been updated